backend/drm/drm.c

```diff
@@ -109,8 +109,15 @@ static bool add_plane(struct wlr_drm_backend *drm,
 	for (size_t j = 0; j < drm_plane->count_formats; ++j) {
 		uint32_t fmt = drm_plane->formats[j];
 
-		if (fmt == DRM_FORMAT_ARGB8888) {
-			// Prefer formats with alpha channel
+		if (fmt == DRM_FORMAT_ARGB2101010) {
+			// Prefer formats with 10 bit color
```

Comment:
I've fixed the typo and collapsed the if blocks. I'm not sure what the ordering is. From skimming the code, it might be related to `formats` in `render/gles2/pixel_format.c`. The order added may be from the order added with `wlr_drm_format_set_add()`. For all I know, the ordering is related to the numbered order of monitors. If that's the case, it would be fine since some monitors could be set to 10-bit and others to 8-bit color. I'd need to read quite a bit more to see how everything is fitting together with `drm_plane->formats` though.